capstone/scripts/convert_s3.py

```diff
@@ -139,29 +140,41 @@ def export_cases_to_s3(bucket: str, redacted: bool, reporter_id: str) -> tuple:
 
     # get in-scope volumes with volume numbers in each reporter
     subset_volumes_metadata = ""
-    for volume in (
-        reporter.volumes.exclude(volume_number=None)
-        .exclude(volume_number="")
-        .exclude(out_of_scope=True)
-    ):
-        # export volume metadata/cases
-        volume_metadata = export_cases_by_volume(
-            volume, reporter_prefix, bucket, redacted
+
+    job = group(
+        export_cases_by_volume.s(
```

Comment:
TIL [`s`](https://docs.celeryq.dev/en/stable/reference/celery.app.task.html#celery.app.task.Task.s) and [signatures](https://docs.celeryq.dev/en/stable/userguide/canvas.html#signatures).

Pretty nice!